src/component/ArticleList.js

```diff
@@ -1,20 +1,45 @@
 import { useEffect, useState } from "react";
-
+import { getArticles } from "../utils/api"
 
 import ItemRow from "./ItemRow";
 import Nav from "./nav";
 
 
-function ArticleList({articleList}) {
+
+function ArticleList() {
 
     const [isLoading, setIsLoading] = useState(true);
+    const [sortBy, setSortBy] = useState('')
+    const [articleList, setArticleList] = useState([]);
+  
+  
+    useEffect(() => {
+      setIsLoading(true);
+      getArticles(sortBy).then((articleListFromApi) => {
+        setArticleList(articleListFromApi);
+        setIsLoading(false);
+      });
+    }, [sortBy]);
+  
 
 return(
     <main>
       {/* {isLoading ? (
```

Comment:
This is loading is commented out, it would be good to bring this back in to indicate to users on a slow connection that something is happening